build.sbt

```diff
@@ -12,7 +12,7 @@ lazy val `sbt-freestyle` = project
   .in(file("."))
   .settings(name := "sbt-freestyle")
   .settings(Seq(
-    addSbtPlugin("com.47deg" % "sbt-org-policies" % "0.7.2"),
+    addSbtPlugin("com.47deg" % "sbt-org-policies" % "0.7.5"),
```

Comment:
I'm not fond of having to change the version number in 2 different places. Thoughts on having a similar syntax(as dependencies) for sbt plugins?
Something like:
`%("sbt-org-policies")`, `addSbtPlugin(%("sbt-org-policies")`, or something else?

Reply:
Sounds good.

I'm good with `addSbtPlugin(%("sbt-org-policies")`, since `%` or `%%` returns the ModuleID you want to use (as a plugin or standard dependency), makes sense?

Reply:
👍   Currently `%` and `%%` only look for libraries(not sbt-plugins).

I'm leaning towards adding another parameter `isSbtPlugin: Boolean = false` to the `%` and `%%` functions and switching between `libs` or `allPlugins` based on it.

https://github.com/47deg/sbt-org-policies/blob/master/src/main/scala/sbtorgpolicies/settings/dependencies.scala#L58
https://github.com/47deg/sbt-org-policies/blob/master/core/src/main/scala/sbtorgpolicies/libraries.scala#L327

Reply:
👍